app/tests/src/test/resources/setup.sh

```diff
@@ -3,44 +3,29 @@
 #   External Environment Variables
 #
 #1. KUBERNETES_NAMESPACE:                   The target namespace.
-#2. SYNDESIS_TEMPLATE_TYPE:                 The template type to use (e.g. syndesis, syndesis-restricted, syndesis-ephemeral-restricted.
-#3. SYNDESIS_TEMPLATE_URL:                  The full url to the templates to use.
-#4. OPENSHIFT_TEMPLATE_FROM_WORKSPACE:      Flag to enable reading the template from the ${WORKSPACE} environment variable.
-#5. OPENSHIFT_TEMPLATES_FROM_GITHUB_COMMIT: The sha of the actual commit to use. Meant to be used for pull request validation (currently unused?).
+#2. OPENSHIFT_MASTER                        The openshift master url.
+#3. DEMO_DATA_ENABLED                       Demo data flag.
+#4. MINISHIFT_ENABLED                       Minishift flag.
 #
 #   Template Parameter Values:
-#
 #   OPENSHIFT_MASTER
 
-#Configure the SYNDESIS_TEMPLATE_TYPE
-if [ -z "${SYNDESIS_TEMPLATE_TYPE}" ]; then
-    SYNDESIS_TEMPLATE_TYPE="syndesis-ci"
-fi
-
 if [ -z "${KUBERNETES_NAMESPACE}" ]; then
     KUBERNETES_NAMESPACE="$(oc project -q)"
 fi
-
-#Configure the SYNDESIS_TEMPLATE_URL
-if [ -n "${SYNDESIS_TEMPLATE_URL}" ]; then
-    true
-elif [ -n "${OPENSHIFT_TEMPLATE_FROM_WORKSPACE}" ]; then
-    SYNDESIS_TEMPLATE_URL="${WORKSPACE}/${SYNDESIS_TEMPLATE_TYPE}.yml"
-elif [ -n "${OPENSHIFT_TEMPLATES_FROM_GITHUB_COMMIT}" ]; then
-    SYNDESIS_TEMPLATE_URL="https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/${OPENSHIFT_TEMPLATES_GIT_COMMIT}/${SYNDESIS_TEMPLATE_TYPE}.yml"
-else
-    SYNDESIS_TEMPLATE_URL="https://raw.githubusercontent.com/syndesisio/syndesis-openshift-templates/master/${SYNDESIS_TEMPLATE_TYPE}.yml"
-fi
-
-#Configure OPENSHIFT_MASTER
 if [ -z "${OPENSHIFT_MASTER}" ]; then
     OPENSHIFT_MASTER="$(oc whoami --show-server)"
 fi
-
 if [ -z "${DEMO_DATA_ENABLED}" ]; then
     DEMO_DATA_ENABLED=false
 fi
 
+SYNDESIS_TEMPLATE_TYPE="syndesis-restricted"
```

Comment:
So is it then that we do not need syndesis-ci.yml anymore ? If so, could we remove this from deploy/, too ?

Reply:
Actually I'm really tempted for now to also remove the non restricted variants, and renaming the -restricted to without -restricted.yml. So:

* Deleting syndesis-ci.yml
* Removing syndesis.yml and syndesis-dev.yml
* Moving syndesis-restricted.yml --> syndesis.yml and syndesis-dev-restricted.yml --> syndesis-dev.yml

At the end, we then only have two templates left.

wdyt ? Would this something you could try to adopt ? It's hard to say what references all need to be updated, but as we are on a monorepo a plain text search could be enough.

Reply:
The only real purpose of the `ci` flavor, was to remove readiness/liveness probes and also remove other features that proved to be problematic in the past.

Since things seem to run, as is right now (which is also aligned with how the team runs syndesis), I think that we can scrap the `ci` flavor. Want me to do it as part of this pr?

Now regarding dumping the rest of the templates, does anyone even use the non-restred ones?

Reply:
Yes, that's my point: No one used the non-restricted templates, never seen this in the wild. So happy to rip it, too, but this should go probably into a different PR.